examples/ct/handlers.go

```diff
@@ -436,8 +450,11 @@ func getEntries(c LogContext, w http.ResponseWriter, r *http.Request) (int, erro
 	defer cancel()
 
 	rsp, err := c.rpcClient.GetLeavesByIndex(ctx, &req)
-	if err != nil || !rpcStatusOK(rsp.GetStatus()) {
-		return http.StatusInternalServerError, fmt.Errorf("get-entries: RPC failed, possible extra info: %v", err)
+	if err != nil {
+		return http.StatusInternalServerError, fmt.Errorf("backend GetLeavesByIndex request failed: %v", err)
+	}
+	if !rpcStatusOK(rsp.GetStatus()) {
+		return http.StatusInternalServerError, fmt.Errorf("backend GetLeavesByIndex request failed, status=%v", rsp.GetStatus())
 	}
 
 	// Apply additional checks on the response to make sure we got a contiguous leaf range.
```

Comment:
Not sure I follow -- we do have a response, in `rsp`.  I've updated the comments to try to distinguish between the RPC response and the HTTP response.